### PR TITLE
Chore/kill bower

### PR DESCRIPTION
**Pull request by JeremyDec**
## Migration from bower to yarn
MFRWW-577
Replacement of bower by yarn : 

- bower packages are now taken directly from npm (some packages are still obtained through bower urls), all dependancies in package.json
- wiredep tasks replaced by ones in injector (GruntFile). 
- no more package-lock.json, bower.json and bowerrc files, only yarn.lock file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "sms"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=0.10.0",
```
Comment (FredericEspiau) [inline package.json]: `"node": ">=8"`

**Comment by ovh-cds** [comment]
CDS Report build-yarn#2.0

* Stage 1
  * deps ✔ 

* Stage 2
  * build ✘  
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#6050.0

* init
  * deps ✘  

* build

**Comment by ovh-cds** [comment]
CDS Report empty#6050.0

**Comment by ovh-cds** [comment]
CDS Report build#6050.1

* init
  * deps ✘  

* build

**Comment by ovh-cds** [comment]
CDS Report build#6050.2

* init
  * deps ✘  

* build

**Comment by ovh-cds** [comment]
CDS Report build#6051.0 ✘  
* init
  * deps ✘

**Comment by ovh-cds** [comment]
CDS Report build-yarn#2.1 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✘  
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#6054.0 ✘  
* init
  * deps ✘

**Comment by ovh-cds** [comment]
CDS Report build-yarn#4.4 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build-yarn#4.9 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#6072.0 ✘  
* init
  * deps ✘

**Pull request merged**